protobuf/vm.proto

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Can we add targets in Makefile for protobuf generation?

Reply:
Handled. Added vm-protobuf as target